### PR TITLE
corrects link to getting started tutorial in main readme

### PR DESCRIPTION
**Pull request by nellshamrell**
I was looking for the getting started tutorial for reference and thought I'd correct the link while I'm at it :)

Signed-off-by: Nell Shamrell <nellshamrell@gmail.com>

### CODE REVIEW

**Comment by ncs-alane** [comment]
@nellshamrell: can you rebase your branch and pull request against ncs-alane-0.5.0, then add "Broken reference to the Getting Started guide" to the "Fixed" list of 0.5.0 in the CHANGELOG?

**Comment by nellshamrell** [comment]
Can do!

**Comment by nellshamrell** [comment]
Done!

File: CHANGELOG.md [hunk 1]
```diff
@@ -159,6 +159,8 @@ major version 0.y.z will maintain a stable public interface.
 * Clarify the gem installation instructions in the README (thanks
   [@nictrix])
 
+* Broken reference to the Getting Started guide
```
Comment (ncs-alane) [inline CHANGELOG.md]: This needs to be under the 0.6.0 release, which I realized was not yet in the file. 😆

**Comment by nellshamrell** [comment]
Corrections made :D

**Comment by ncs-alane** [comment]
Thank you for your patience!

**Pull request merged**